packages/nimble-components/src/table/specs/editable-table-ixd.md

```diff
@@ -0,0 +1,103 @@
+# Nimble Editable Table (IxD)
+
+## Overview
+
+Nimble table support for interactive cell editing.
+
+### Background
+
+-   [Figma worksheet](https://www.figma.com/design/r2yGNQNVFdE7cBO9CyHmQx/Nimble---IxD?node-id=1221-36463)
+-   [Input table columns (select, text field, etc) #1190](https://github.com/ni/nimble/issues/1190)
+
+## Usage
+
+**When to use:**
+
+-   When users need to quickly and efficiently update data directly within the table.
+-   When inline editing can improve the user experience by reducing the need for separate forms or dialogs.
+-   When the table is used in scenarios where data accuracy can be ensured through validation and error handling mechanisms.
+
+**When not to use:**
+
+-   When data integrity is critical and the table should not be modified by users.
+-   When the table is used for displaying static or read-only data.
+-   When the user interface requires a more complex data entry form.
+
+## Anatomy
+
+![Editable table anatomy](./spec-images/editable-cell-anatomy.png)
+
+| Element               | Description                                                              |
+| --------------------- | ------------------------------------------------------------------------ |
+| Static cell value     | Non-editable display of cell value                                       |
+| Action button         | Button to open cell context menu. Appears on row hover or keyboard focus |
+| Editable cell control | Text or numeric control that supports entering or editing cell data      |
+
+## Behavior
```

Comment:
What's the scope of "editability"? 

1. Do apps mark the entire table as editable? Specific rows? Specific columns? Specific cells?
2. Does every column type support editing? For pretty much everything except text columns that opens the door to questions about what the editor should look like and what kind of validation is available.

Reply:
Specific columns and we'll need to start with text and numeric, and add select and checkbox later. We shouldn't need to support every column type (at least right away.)